acceptance-tests/helpers/brokers/create.go

```diff
@@ -63,8 +64,18 @@ func WithSourceDir(dir string) Option {
 		b.dir = dir
 	}
 }
+func WithConfig(config map[string]interface{}, dir string) {
 
+	bytes, err := json.Marshal(config)
+	if err != nil {
+		panic(err)
+	}
+
+	os.WriteFile(fmt.Sprintf("%s/config.yml", dir), bytes, 0666)
```

Comment:
issue: there's string interpolation of file names and a lack of error checking. This should be:
```go
Expect(os.WriteFile(filepath.Join(dir, "config.yml", bytes, 0666))).To(Succeed())
```